includes/class-admin-settings.php

```diff
@@ -671,6 +712,9 @@ public function add_settings_field_callback( $args = [] ) {
 					?>
 				</select>
 				<p>
+					<label for="aspireupdate-settings-field-<?php echo esc_attr( $id ); ?>_other">
+						<?php esc_html_e( 'Enter a custom API host. Ensure that it starts with http:// or https://', 'aspireupdate' ); ?>
+					</label><br>
 					<input
 						type="url"
 						data-pattern="(https?)://.*"
```

Comment:
Per comment above, while I think that the instructions should change, I *don't* necessarily think that means that the validation needs to change.